cloud/filestore/config/vfs.proto

```diff
@@ -36,4 +36,12 @@ message TVFSConfig
 
     // Limits max requests allowed at a time.
     optional uint32 MaxBackground = 12;
+
+    // Path to the directory where HandleOpsQueue is stored for all sessions.
+    // Will create a directory with the filesystemId
+    // and a subdirectory with the sessionId if not present.
+    optional string HandleOpsQueuePath = 13;
+
+    // Max HandleOpsQueue size in bytes.
+    optional uint32 HandleOpsQueueSize = 14;
```

Comment:
I thought it would be better to set the queue size equal to the file size to have better control over the memory usage on the host with these queues. Maybe better to rename this parameter to HandleOpsQueueFileSizePerSession?